app/packages/components/src/components/ModalBase/ModalBase.tsx

```diff
@@ -101,13 +101,14 @@ const ModalBase: React.FC<ModalBaseProps> = ({
 
   const modalButtonView: ModalButtonView = {
     variant: props?.variant || "outlined",
-    label: props?.label || "Open Modal",
+    label: props?.label || "",
     componentsProps: {
       button: {
         sx: {
           height: props?.height || "100%",
           width: props?.width || "100%",
-          padding: 1,
+          padding: props?.padding || 1,
+          minWidth: 0,
```

Comment:
why do you need this?

Reply:
Yeah, it was a weird edge case. So basically when I try to render ModalView at the top level button, if it is only an Icon then it inherits both the width of normal Button + Icon which was causing a generic Arrow for example to have way too much space & padding

Adding minWidth=0 while also giving flexibility to padding gives the best of both worlds ie., no extra icon padding and can still define additional padding to supplant minWidth on a regular button look